### ISSUE TITLE
Ensure compiles / works on Windows

### ISSUE THREAD

**Issue by heartsucker** [heartsucker/rust-tuf#23]

**Comment by heartsucker** [comment]
Progress on branch `windows-support`.

**Comment by heartsucker** [comment]
Omg windows rust development is impossible. Leaving this out until someone claims it.

**Comment by briansmith** [comment]
What trouble are you having? I do most of my Rust development on Windows. As long as things don't depend on OpenSSL and similar libraries, they usually work.

**Comment by heartsucker** [comment]
I installed VistualStudio in a VM with the C++ package, installed rust via `rustup.exe`, and then started getting linker errors (`advapi31.lib`). I found some issues on the Rust repo that made it sound like Rust isn't plug-and-play ready for windows, and since I've never programmed on Windows, I'm leaving this for someone else (for now). Right now the only thing I'm having trouble with is converting URLs into WIndows paths (which I could probably do away with entirely, but a URL seemed like a simple way to address both HTTP and FS resources).
